scipy/__config__.py.in

```diff
@@ -109,6 +109,9 @@ CONFIG = _cleanup(
             "path": r"@PYTHON_PATH@",
             "version": "@PYTHON_VERSION@",
         },
+        "Coverage Information": {
+            "GCOV": "@GCOV_FLAG@"
+        }
```

Comment:
I'd prefer not to add this to this data structure, since it's not relevant to users and `show()` is a public function. Isn't it easier anyway to for example check that a build subdirectory contains a `.gcno` file? E.g.:
```
$ ls build/scipy/cluster/_vq.cpython-312-x86_64-linux-gnu.so.p
meson-generated__vq.c.gcno  meson-generated__vq.c.o  _vq.c
```
If the build used coverage, walking a single subdirectory and checking for `any(exists('*.gcno'))` should be enough.